sys_unix_amd64.s

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
After the CGO_ENABLED=1 fix put this file back into sys_amd64.s with an updated build tag to include linux